http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/leonardoServiceContractModel.scala

```diff
@@ -1,205 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo
```

Comment:
was this just a file rename?

Reply:
Yeah I think I capitalized the filename to be consistent

Reply:
Faces: we'll lowercase files without a parent "type" -- e.g. models files